src/rdo.rs

```diff
@@ -1196,6 +1196,7 @@ pub fn rdo_cfl_alpha<T: Pixel>(
 ) -> Option<CFLParams> {
   let PlaneConfig { xdec, ydec, .. } = ts.input.planes[1].cfg;
   let uv_tx_size = bsize.largest_chroma_tx_size(xdec, ydec);
+  debug_assert!(bsize.subsampled_size(xdec, ydec) == uv_tx_size.block_size());
```

Comment:
Was CfL enabled for a partition smaller than a certain size, like 32x32? Forgot that. 
(Because, I aware the cases where uv_tx_size can be smaller than subsampled bsize in the future,
such as with 128x128 SB added, and with that 4:4:4 chroma format will have 128x128 chroma bsize that can not have 128x128 tx, since the largest tx size is 64x64)

Reply:
Oh, apparently cfl_allowed( ) says "self <= BlockSize::BLOCK_32X32"!

Reply:
Also, I realized that the above debug_assert( ) seems indeed started from me! :)

Reply:
Keeping this as it documents one of the core assumptions of AV1 CfL.